web/src/settings_notifications.ts

```diff
@@ -86,6 +87,7 @@ function rerender_ui(): void {
         $unmatched_streams_table.css("display", "table-row-group");
     }
     update_desktop_notification_banner();
+    settings_banner.set_up_mobile_push_banner();
```

Comment:
I think there is no need to call this here, since showing the banner depends on `realm.realm_push_notifications_enabled` field and we do not live-update on changing that field. This can also be called in set_up.

Reply:
Also, the banner is not shown in "Default user settings" panel if I do not open the "Notifications" panel at least once.